wpilibNewCommands/src/main/native/include/frc2/command/Command.h

```diff
@@ -269,6 +269,16 @@ class Command : public wpi::Sendable, public wpi::SendableHelper<Command> {
   [[nodiscard]]
   CommandPtr Repeatedly() &&;
 
+  /**
+   * Decorates this command to run repeatedly until the given count is reached
+   * or is interrupted. The decorated command can still be canceled.
```

Comment:
This would mean we would neet to also change the documentation on the `.repeating()` function.

IMO, it would be better if this PR didn't have any line removals

Reply:
> This would mean we would neet to also change the documentation on the `.repeating()` function.

How so? I'm just talking about changing the `repeatedly(int)` doc comments.

Also, the wording that would most match the `repeatedly()` doc comment (in both Java and C++) is "Decorates this command to run repeatedly, restarting it when it ends, until this command is run the specified number of times or is interrupted. The decorated command can still be canceled."